ansible/playbooks/aix.yml

```diff
@@ -280,21 +299,28 @@
           register: java9
           tags: java9
 
-        - name: Create /usr/java9_64 if it does not exist
-          file:
-            path: /usr/java9_64
-            state: directory
-          when: java9.stat.isdir is not defined
-          tags: java9
-
         - name: Transfer and Extract AdoptOpenJDK 9
           unarchive:
             src: https://api.adoptopenjdk.net/v2/binary/releases/openjdk9?openjdk_impl={{ bootjdk }}&os=aix&arch=ppc64&release=latest&type=jdk
-            dest: /usr/java9_64
+            dest: /usr
             remote_src: yes
           when: java9.stat.isdir is not defined
           tags: java9
 
+        - name: Find java9 directory
+          find:
+            file_type: directory
+            paths: /usr
+            patterns: 'jdk9*'
```

Comment:
I'm not sure that's the correct pattern to pick up the contents of the java9+ tarball